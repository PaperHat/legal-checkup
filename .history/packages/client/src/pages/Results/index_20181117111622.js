import React from 'react'
import Layout from '../../components/Layout'
<<<<<<< HEAD
import ResultsHero from '../../components/Results'
=======
import DesktopRedoCheckup from '../../components/DesktopRedoCheckup'
>>>>>>> techdebt/convert_to_lerna_and_add_server

export default function Results () {
  return (
    <Layout>
<<<<<<< HEAD
      <ResultsHero />
=======
      <p>
        This is the <strong>Results</strong> page
      </p>
      <div>
        <DesktopRedoCheckup />
      </div>
>>>>>>> techdebt/convert_to_lerna_and_add_server
    </Layout>
  )
}
