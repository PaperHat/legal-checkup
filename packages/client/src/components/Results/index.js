import * as React from 'react'
import { connect } from 'react-redux'

import { getYesAnswerCount } from '../../state/selectors'

function ResultsHero ({ yesCount }) {
  return (
    <>
      <p>
        {yesCount > 0
          ? `Based on your answers, a lawyer may be able to help you with...`
          : `Based on your answers, it looks like you're in good legal health! 
        But if you have a problem, or something worrying you that wasn't covered by the questions 
        in this check-up, do come talk to us at Community Law`}
      </p>
      {yesCount > 0 ? <a href='#'>expand</a> : null}
      {yesCount > 0
        ? <p>
            You can get free legal advice about these (or any other problem) from
            your local Community Law Centre.
        </p>
        : null}
    </>
  )
}

function mapStateToProps ({ checkup }) {
  return {
    yesCount: getYesAnswerCount(checkup)
  }
}

export default connect(mapStateToProps)(ResultsHero)
