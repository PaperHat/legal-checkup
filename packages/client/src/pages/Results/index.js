import React from 'react'
import Layout from '../../components/Layout'
import RedoCheckup from '../../components/RedoCheckup'

export default function Results () {
  return (
    <Layout>
      <p>
        This is the <strong>Results</strong> page
      </p>
      <div>
        <RedoCheckup />
      </div>
    </Layout>
  )
}
