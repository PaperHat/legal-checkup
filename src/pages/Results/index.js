import Layout from '@containers/Layout'
import React from 'react'

export default function Results () {
  return (
    <Layout>
      <p>
        This is the <strong>Results</strong> page
      </p>
    </Layout>
  )
}
