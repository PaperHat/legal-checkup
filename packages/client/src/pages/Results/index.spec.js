import React from 'react'
import { shallow } from 'enzyme'

import Results from './'

describe('pages:Results', () => {
  it('renders the Results page', () => {
    expect(toJson(shallow(<Results />))).toMatchSnapshot()
  })
})
