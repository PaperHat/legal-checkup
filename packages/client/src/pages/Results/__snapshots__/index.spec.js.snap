// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages:Results renders the Results page 1`] = `
<Layout>
  <Connect(ResultsHero) />
  <p>
    This is the 
    <strong>
      Results
    </strong>
     page
  </p>
  <div>
    <Connect(DesktopRedoCheckup) />
  </div>
</Layout>
`;
